jekyll/_cci2/orb-intro.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: classic-docs
+title: "Orbs Introduction"
+short-title: "Orbs Introduction"
+description: "Starting point for CircleCI orbs"
+categories: [getting-started]
+order: 1
+---
+
+CircleCI provides certified and tested orbs, along with 3rd-party orbs authored by CircleCI partners. It is best practice to first evaluate whether any of these existing orbs will help you in your configuration workflow. Refer to the [CircleCI Orbs Registry](https://circleci.com/) for the complete list of available Orbs. 
+
+### Importing an Existing Orb
+
+If you wish to import an existing orb, it would be similar to the example shown below.
+
+```
+orbs:
+  slack: circleci/slack@0.1.0
+  heroku: circleci/heroku@1.0.0
+```
+
+In the above example, two orbs would be made available to you (slack & heroku), one for each key in the map.
+
+Because the values of the above keys under `orbs` are all scalar values they are assumed to be imports based on the orb ref format of `${NAMESPACE}/${ORB_NAME}@${VERSION}`. 
+
+### Authoring Your Own Orb
+
+If you find that there are no existing orbs that meet your needs, you may author your own orb to meet your specific environment or configuration requirements. Although this is more time-consuming than using the import feature, authoring your own orb enables you to create an orb that is specific to your organization.
```

Comment:
This is an excellent opportunity to make clear that all orbs are open orbs and therefore world-readable. The phrase" specific to your organization" may send the message that private orbs are a thing, but they aren't yet.

Reply:
right, I changed this to "authoring your own orb enables you to create a world-readable orb for sharing your configuration"